docs/faqs/related_projects.md

```diff
@@ -41,9 +41,9 @@ approach with a strong separation of concerns, that enables applications to
 easily and safely consume the infrastructure they need, using any tool that
 works with the Kubernetes API. GCP Config Connector is closed-source.
 
-## AWS Service Operator
+## AWS Controllers for Kubernetes
 
-The [AWS Service Operator] is a recent project that implements a set of
+The [AWS Controllers for Kubernetes] is a recent project that implements a set of
 Kubernetes controllers that are able to provision managed services in AWS. It
 defines a set of CRDs for managed services like DynamoDB, and controllers that
 can provision them via AWS CloudFormation. It is similar to Crossplane in that
```

Comment:
Thanks for the update @joebowbeer! I think the AWS CloudFormation part is no longer accurate for ACK, so I would encourage removing this if we want to make this change. Overall I think this doc needs a larger update, but it would be good to go ahead and update this comparison :)
```suggestion
can provision them. It is similar to Crossplane in that
```